css/style.css

```diff
@@ -0,0 +1,91 @@
+body {
+  font-family: sans-serif;
+  font-weight: 400;
+}
+
+.justify-content-end {
+  font-size: 25px;
+  background-color: #be3144;
+  height: 80px;
+}
+
+.justify-content-end li {
+  margin-top: 10px;
+  margin-right: 40px;
+  color: black;
+  font-size: 26px;
+}
+
+.justify-content-end li a {
+  color: white;
+}
```

Comment:
I think you did a really great job here but please if you can add hover to the menu because it will look more professional.